SheepDash/core/src/com/missionbit/game/sprites/Farmer.java

```diff
@@ -0,0 +1,61 @@
+package com.missionbit.game.sprites;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureRegion;
+import com.badlogic.gdx.math.Rectangle;
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.math.Vector3;
+
+/**
+ * Created by missionbit on 6/29/17.
+ */
+
+public class Farmer {
+    private Texture farmer;
+    private Animation farmerAnimation;
+    private Vector3 position;
+    private Vector3 velocity;
+    private Rectangle boundsFarmer;
+    private Vector2 posFarmer;
+    private static final int MOVEMENT = 83;
+
+    public Farmer(int x, int y){
+        farmer = new Texture("Farmer on a Tractor.png");
```

Comment:
You also need to commit this file, "Farmer on a Tractor.png" 
Otherwise the code doesn't compile